lib/openfeature/sdk/evaluation/context.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+module OpenFeature
+  module SDK
+    module Evaluation
+      # A container for arbitrary contextual data that can be used as a basis for dynamic evaluation
+      class Context < SimpleDelegator
+
+        def initialize(context = Concurrent::Hash.new({}))
```

Comment:
This is part of the [concurrent-ruby](https://github.com/ruby-concurrency/concurrent-ruby) and it's not a dependency. @josecolella included this dependency initially, but we ended up removing it for simplificy.